framework/src/modules/chain/synchronizer/utils.js

```diff
@@ -78,31 +94,32 @@ const restoreBlocksUponStartup = async (
 
 	if (inDifferentChain) {
 		// In case fork status is DIFFERENT_CHAIN - try to apply blocks from temp_block table
-		await restoreBlocks(blocksModule, processorModule, {});
+		await restoreBlocks(blocksModule, processorModule);
 	} else {
 		// Not different chain - Delete remaining blocks from temp_block table
-		await storageModule.entities.TempBlock.truncate();
+		await clearBlocksTempTable(storageModule);
```

Comment:
I think in both cases temp block should be cleared?

Reply:
@shuse2 `restoreBlocks` function already removes the blocks from temp table